ui/pages/send/send-content/send-from/send-from.component.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
+import { compose } from 'redux';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+import ArrowIcon from '../../../../components/ui/icon/arrow-icon.component';
+import UserPreferencedCurrencyDisplay from '../../../../components/app/user-preferenced-currency-display';
+
+import {
+  getMetaMaskAccounts,
+  getMetaMaskAccountsOrdered,
+} from '../../../../selectors';
+
+function mapStateToProps(state) {
+  return {
+    accountsList: getMetaMaskAccountsOrdered(state),
+    accounts: getMetaMaskAccounts(state),
+    selectedAddress: state.metamask.selectedAddress,
+  };
+}
+
+function SendFrom({ accounts, selectedAddress, accountsList }) {
+  const balanceValue = accounts[selectedAddress]
+    ? accounts[selectedAddress].balance
+    : '';
+  const t = useI18nContext();
+  return (
+    <div className={classnames('send-v2__from-wrap')}>
+      <div className="send-v2__from-title">from</div>
+      <div className="send-v2__from-cont">
+        <div className="send-v2__from-cont-img">pic</div>
```

Comment:
Should be `Identicon` for now

Reply:
fixed in bd5a9d52dfae71fc07e4aa91d9c7d3d7ede5c444

Reply:
- Verified